Lake/Config/Workspace.lean

```diff
@@ -30,6 +31,8 @@ structure Workspace : Type where
   packageFacetConfigs : DNameMap PackageFacetConfig
   /-- Name-configuration map of library facets defined in the workspace. -/
   libraryFacetConfigs : DNameMap LibraryFacetConfig
+  /-- The verbosity setting for logging messages. -/
+  verbosity : Verbosity := .normal
```

Comment:
Why does the `Workspace` need to store the verbosity setting? From what I can tell all the places you have used this (e.g., `printPaths`) could just get this from the CLI directly. I do not feel this setting belongs here.

Reply:
I had a lot of difficulty threading around global logging context due to all the monad juggling that lake does. I can't remember which part requires this, I'll try removing it and report back.

Reply:
The only issue I hit is that `Lake.exe` does not have the verbosity setting available. Should it be in the `LakeT IO` monad, or an extra parameter? For now I've made it a parameter.

Reply:
Either works for me.